app/src/main/java/com/example/android/roomwordssample/WordDao.kt

```diff
@@ -40,10 +41,7 @@ interface WordDao {
     @Query("SELECT * from word_table ORDER BY word ASC")
     fun getAlphabetizedWords(): LiveData<List<Word>>
 
-    // We do not need a conflict strategy, because the word is our primary key, and you cannot
```

Comment:
Could you either: 
1. Make this change on the Java branch as well
2. Revert this change in this PR

Reply:
Done, see https://github.com/googlecodelabs/android-room-with-a-view/pull/18